Formula/fontforge.rb

```diff
@@ -101,6 +101,18 @@ def install
     end
   end
 
+  def caveats
+    <<-EOS.undent
```

Comment:
@MikeMcQuaid Essentially, if you want to change a majority of caveats to match your preferred style, that's fine, I don't have an objection. My objection is a slow drift where nobody really knows what to use and we get an inconsistent splattering of style.